conda_smithy/configure_feedstock.py

```diff
@@ -1483,6 +1505,7 @@ def _load_forge_config(forge_dir, exclusive_config_file):
         "conda_forge_output_validation": False,
         "private_upload": False,
         "secrets": [],
+        "silenced_vars": ["python_impl"],
```

Comment:
I'd rather not totally silence things but instead shorten them.

Reply:
Also I’m pretty sure silencing things will break builds in some cases.

Reply:
> Also I’m pretty sure silencing things will break builds in some cases.

How would it break builds in your opinion? That's why the code checks whether the variable is needed to distinguish builds.

Reply:
> I'd rather not totally silence things but instead shorten them.

I don't understand what you mean.

Reply:
I didn’t realize the code checked to make sure things were unique. 

We can shorten python to py for example.

Reply:
Sure but in the case of `python` I think `x.y_cpython/pypy` should leave no doubt.

Reply:
Ok, and throw away leading `lib` and remove some key names that are currently not helpful such as `target_platform`.

Reply:
Target platform is quite helpful imho

Reply:
`target_platform` is repeated, so it's not needed. For eg: https://github.com/conda-forge/r-pcapp-feedstock/tree/a1c2f4145a10eb80ebbf9bb84b6fc12982261935/.ci_support

Reply:
What about a feedstock with aarch 64?

Reply:
Ahhh never mind. It is there at the front. I’m ok removing it. Thanks for taking the time to explain that @isuruf.

Reply:
And thanks @saraedum for your persistence here too!